WooCommerce/Classes/ViewRelated/Products/Edit Product/Edit External Link/ProductExternalLinkViewController.swift

```diff
@@ -122,24 +127,29 @@ private extension ProductExternalLinkViewController {
     }
 
     func configureExternalURL(cell: TitleAndTextFieldTableViewCell) {
-        let title = NSLocalizedString("Product URL", comment: "Title of the text field for editing the external URL for an external/affiliate product")
-        let placeholder = NSLocalizedString("Enter URL",
-                                            comment: "Placeholder of the text field for editing the external URL for an external/affiliate product")
-        let viewModel = TitleAndTextFieldTableViewCell.ViewModel(title: title,
+        let title = Strings.productURLTitle
+        let placeholder = Strings.enterURLPlaceholder
+        var viewModel = TitleAndTextFieldTableViewCell.ViewModel(title: title,
                                                                  text: externalURL,
                                                                  placeholder: placeholder,
                                                                  keyboardType: .URL,
                                                                  textFieldAlignment: .trailing,
                                                                  onTextChange: { [weak self] text in
                                                                     self?.externalURL = text
+                                                                    if text?.isValidURL() == true {
+                                                                        self?.hideError()
+                                                                    }
+                                                                    else {
```

Comment:
nit: we usually have `} else {` on the same line :)

Reply:
I forgot always the same thing 😱